### DISCUSSION TITLE
How to simulate the effect of food on drugs?

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#787] [Questions & Problems] [answered]
Dear to all,
I intend to simulate the effect of food on the pharmacokinetics of a drug, which may require modification of certain physiological and anatomical parameters, could you provide some literatures for me? Can this process be achieved in the PK-SIM?
Sincerely！

**Comment by StephanSchaller** [comment]
Yes, this is possible. Please refer to the documentation:

https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-events#definition-of-new-events-in-pk-sim-r

**Comment by Wenjing1018** [comment]
@StephanSchaller，thank you for your reply.
I wonder if the parameters caused by food will change accordingly after adding meals to the event? Or do I need to manually modify the physiological and anatomical parameters?

**Comment by StephanSchaller** [comment]
Which parameters are you referring to?

For the meal, a few parameters such as gastric emptying time are changed and return back to normal after some time:

"_For the various meals provided, the pH in the stomach and the gastric emptying function will change significantly. Irrespective of the meal chosen, the gastric pH will be increased to 5.5 and then decay exponentially. The rate of gastric emptying, which controls the transport of the drug to the absorption sites in the intestine, will change according to a function that is based on the Weibull equation. The Weibull function was parameterized based on about 100 datasets for gastric emptying profiles in humans following ingestion of various meals [81]._"

**Comment by Wenjing1018** [comment]
@StephanSchaller，
the changes in parameters brought by food, in addition to changes in gastric PH、 gastric emptying time, may also lead to changes in parameters such as stomach volume, duodenum PH、 liver blood flow and so on. I wonder if these parameters need to be obtained from the literature?If so, could you provide some literature that involved in change of parameters?
Thank you very much！
Sincerely!

**Comment by AndreDlm** [comment]
In addition to changes in gastric pH and emptying time, stomach lumen volume also increases (the fluid volume of the meal is added). 
Other changes e.g. in liver blood flow are not accounted for, but you could manually change them if you have good reason to believe that these parameter should be different

**Comment by Wenjing1018** [comment]
@AndreDlm，Your reply is very helpful to me！
I'd like to know the change of gastric pH and gastric emptying time in details ,I tried to find a reference source for the following sentence, but failed.
"For the various meals provided, the pH in the stomach and the gastric emptying function will change significantly. Irrespective of the meal chosen, the gastric pH will be increased to 5.5 and then decay exponentially. The rate of gastric emptying, which controls the transport of the drug to the absorption sites in the intestine, will change according to a function that is based on the Weibull equation. The Weibull function was parameterized based on about 100 datasets for gastric emptying profiles in humans following ingestion of various meals [81]."

**Comment by AndreDlm** [comment]
When you export your model to MoBi, you will find the Weibull functions to describe the change in gastric pH and transit time in the `Events` building block under the defined food intake, e.g.:

![image](https://user-images.githubusercontent.com/26541603/98573516-4102e380-22b7-11eb-9ffc-c364182ab7c9.png)

**Comment by Wenjing1018** [comment]
Dear AndreDlm
I got it. Thank you for your patient reply!

**Comment by Wenjing1018** [comment]
Dear to all,
Sorry to bother you again！
I want to know what the parameter values in the formula are and what the reference source for the formula is？
![image](https://user-images.githubusercontent.com/63133525/98753399-38b5c000-23ff-11eb-94a0-0c076bb5d92b.png)

**Comment by AndreDlm** [comment]
That's a good question and the introduction of these functions dates long back! 
The food effect on gastric emptying rate is described [here](https://www.page-meeting.org/?abstract=2355). Unfortunately, I am not aware of a pdf of that poster that is available somewhere, but I'll have again a closer look...
However, this poster presents several functions incl. exponential, Hill, Weibull and others that were fitted to experimental data on gastric emptying - if I remember correctly to data reported in this [study](https://pubmed.ncbi.nlm.nih.gov/3699551/). The Weibull function scored best and was therefore chosen to describe gastric emptying rate. 
It was also found that meal energy content and the meal's solid fraction affected gastric emptying rate. The relationship between these factors and T50 (time at which 50% of the initial meal volume has left the stomach) was then analyzed resulting in the current implemented equation where β is calculated from the meal's fraction solid and α from the meal's energy content and β. 
I hope this helps a bit...

**Comment by Wenjing1018** [comment]
@AndreDlm，
Your reply is very helpful to me，I saw in the previous question that the parameter values in the formula can be displayed, and I wonder how to show them in MOBI?[#566](https://github.com/Open-Systems-Pharmacology/Forum/issues/566)
![image](https://user-images.githubusercontent.com/63133525/98806077-9b38ab80-2453-11eb-8a07-79be5b44d4c5.png)
yours sincerely!

**Comment by Wenjing1018** [comment]
Dear to all,
I wonder what is the meaning of start time in the event ?
For example, 
i：drugs were taken with breakfast, what is the value of start time?
ii：Breakfast was served at 8:40 a.m., the drug was taken  at 9:00 a.m.,what is the value of start time?
iii：the drug was taken  at 8：40 a.m.，Breakfast was served at 9:00 a.m.,what is the value of start time?
Sincerely!
![image](https://user-images.githubusercontent.com/63133525/98914423-9df2d980-2503-11eb-8c25-f056c076af4f.png)

**Comment by Fatima-Marok** [comment, marked as answer]
With the start time you can define at which time point in your simulation the event starts. 
I would recommend to simulate the time differences with regards to your administration protocol. If you use an advanced protocol, you can define start times of your drug intake. 
For your examples:
i: Start time event and start time administration both at the same time ( i.e. 0 h)
ii: Start time event at 0 h and start time administration at 0.33 h
iii: Start time administration at 0 h and start time event at 0.33 h

one advice: Keep in mind to time your observed data accordingly, if you change the start of your administration.

I hope this helps!

**Comment by Wenjing1018** [comment]
@Fatima-Marok 
Your answer is very helpful to me，Thank you for your reply！
